README.md

```diff
@@ -342,7 +404,7 @@ const customCategories = {
 
 The values for `integrations` should be an integration's creationName (`integration.creationName`). You can find examples of that by going to `https://cdn.segment.com/v1/projects/<writeKey>/integrations`
 
-#### Example
+##### Custom categories Example
```

Comment:
I think this title has to be as it was before since it is a general example and there are no custom categories.